Resources/public/css/layout.css

```diff
@@ -355,6 +355,10 @@ body.fixed .content-header .navbar.stuck {
     border-radius: 0;
 }
 
+.box .nav-stacked > li {
```

Comment:
As mentioned before, this is a general bug in the nav-stacked element. So why use a special css class?